gunicorn/arbiter.py

```diff
@@ -522,37 +522,40 @@ def reap_workers(self):
                     # A worker was terminated. If the termination reason was
                     # that it could not boot, we'll shut it down to avoid
                     # infinite start/stop cycles.
-                    exitcode = status >> 8
+
+                    exitcode = os.WEXITSTATUS(status) \
```

Comment:
why not reusing the status ?

Reply:
I was trying to find out why I was/am seeing 0 on worker reload, where I was expecting to see 15 for SIGTERM. I was hoping that this might give me more, but clearly 0 stays 0.